pages/common/nnn.md

```diff
@@ -0,0 +1,36 @@
+# nnn
+
+> Interactive terminal file manager and disk usage nalyzer.
+> More information: <https://github.com/jarun/nnn>.
+
+- Open the current directory (or specify one as the first argument):
+
+`nnn`
+
+- Open an existing bookmark:
+
+`nnn -b {{bookmark_name}}`
+
+- Show hidden files:
+
+`nnn -d`
+
+- Start in disk usage analyzer mode:
+
+`nnn -S`
+
+- Start in light mode (fewer details):
+
+`nnn -l`
+
+- Copy selected file path to file_out, or stdout if file_out='-':
+
+`nnn -p {{file_out}}`
```

Comment:
Just say file_path instead of file_out. And maybe change the latter part of the description to - "or stdout if file_path is '-'"

Reply:
I'd suggest the following:

    - Copy the selected file path to a specific output file (stdout if `out_file` is `-`):

    `nnn -p {{file_out}}`

And would also suggest leaving `file_out` as is, since the path being copied is not the one of that file, but the one selected by the user.

Reply:
Example:

![Peek 2019-06-07 12-58](https://user-images.githubusercontent.com/14198070/59099674-14110b00-8924-11e9-87bc-b1bded86cc3b.gif)

Reply:
Oh yeah. Then we should follow the usual convention I think - path/to/output_file. Hows that sound ?

Reply:
@agnivade you're right (edited my comment above just to avoid duplicating code).